configure.ac

```diff
@@ -78,7 +78,8 @@ m4_define([libva_lt_age],
 m4_define([libdrm_version], [2.4])
 
 # Wayland minimum version number
-m4_define([wayland_api_version], [1.0.0])
+# 1.11.0 for wl_proxy_create_wrapper
+m4_define([wayland_api_version], [1.11.0])
```

Comment:
Hm I can see that this is unfortunate, but what are you suggesting then? Keep it broken? There is just no way to realize completely thread-safe behavior prior to 1.11.
Guess the distros will have to update wayland to 1.11 if they want to update libva. If you're compiling libva by yourself already and not using the distro libva packages, I don't really see the problem. It's to be expected that you will also have to build some dependencies if you need the latest version.